staging/src/k8s.io/apiserver/pkg/endpoints/handlers/apply.go

```diff
@@ -92,6 +92,25 @@ func (p *applyPatcher) convertResultToUnversioned(result apply.Result) (runtime.
 	return uoutput, nil
 }
 
+func (p *applyPatcher) saveNewIntent(obj runtime.Object, workflow string) error {
+	// We want to consistently save the intent in JSon.
+	json, err := yaml.YAMLToJSON(p.patchBytes)
```

Comment:
It's confusing that this uses p.patchBytes but convertCurrentVersion takes an input object.

Reply:
Ah right, sorry.